src/stylesheets/_variables.scss

```diff
@@ -4,13 +4,16 @@
   --wedgewood: #4f728e;
   --light-aqua: #4f9c95;
   --grey: rgb(137, 167, 189);
+  --darkmode: #1a1542;
+  --blue: #4366c0;
 
   /* variations */
   --wedgewood__dark: #{darken(#4f728e, 7%)};
 
   /* These are for gradient */
   --wedgewood__gradient: #486988;
   --waterleaf: #a9ebdf;
+  --blue_gradient: #344790;
```

Comment:
In order to be more in line with the BEM naming conventions, we should have this be `blue__gradient`(with the double underscore) or `blueGradient`